bin/node/runtime/src/lib.rs

```diff
@@ -1521,6 +1520,64 @@ impl pallet_office::Config<AssetRegistryOfficeInstance> for Runtime {
 	type WeightInfo = ();
 }
 
+parameter_types! {
+	pub const LLDBridgePalletId: PalletId = PalletId(*b"lldbridg");
+	pub const LLMBridgePalletId: PalletId = PalletId(*b"llmbridg");
+	pub const MaxRelays: u32 = 20;
+	pub const MaxWatchers: u32 = 20;
+	pub const WithdrawalDelay: BlockNumber = 60 * MINUTES;
+
+	// * 66k LLD in single hour (1 burst + hour avg)
+	// * 894k LLD in single day (1 burst + 24*hour avg)
+	pub const LLDRateLimit: (Balance, Balance) = (
+		// max burst and max single withdrawal
+		30_000 * DOLLARS,
+		// decay per block (max average withdrawal rate over infinite time)
+		// 60 LLD * 600 blocks/h = max avg 36k LLD per hour
+		60 * DOLLARS
```

Comment:
likely less, as 1.3 mil is the total issuance, but ok, we can change it for mainnet